packages/react-core/src/components/Alert/examples/Alert.md

```diff
@@ -71,11 +98,142 @@ import { Alert, AlertActionCloseButton, AlertActionLink } from '@patternfly/reac
   </React.Fragment>
 ```
 
-### Inline types
+#### Expandable alerts
+
+An alert can contain additional, hidden information that is made visible by clicking a chevron icon. This information can be expanded and collapsed each time the icon is clicked.
+
+It is not recommended to use an expandable alert within a toast [alert group](/components/alert#alert) because the alert could timeout before users have time to interact with and view the entire alert.
```

Comment:
For me, the first part of the sentence feels like an expandable alert should never be used within a toast alert group. However, the other part (`because the alert could timeout before users have time to interact with and view the entire alert`) is true only if the `timeout` prop is used.

I'm not sure if there are any other reasons why not to use expandable alerts within toast alert group, but if there aren't any, maybe we could be more specific and change the first part to something like this:
```
It is not recommended to use an expandable alert with timeout within a toast...
```
Otherwise, we could use some better reason why not to use this combination.

@mcarrano @thatblindgeye what do you think?

Reply:
Good catch @tompsota! I'd agree with you and Matt about changing the verbiage a bit.

Another issue with using expandable alerts within a toast alert group (or any alert with interactive content that might be necessary for a user to interact with) could be that some users would find it more difficult to interact with these alerts (see [alert accessibility considerations](https://www.patternfly.org/v4/components/alert/design-guidelines#accessibility-considerations) currently on the design guidelines tab). So if important information describing the alert is in that collapsed section of the alert, a user might have to navigate through the entire page before landing on the alerts (assuming they're appended at the end of the document body).